### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by lorenzoworx**
- Used media queries
- Update HTML
- Update CSS
- Add relevant images

### CODE REVIEW

File: CSS/Styles.css [hunk 1]
```diff
@@ -563,5 +563,9 @@ body {
   .connect > a {
     text-decoration: none;
   }
+
+  .desktop {
+    display: none;
+  }
 }
 /* stylelint-enable no-descending-specificity */
```
Comment (blessedjasonmwanza) [inline CSS/Styles.css]: - [ ] **Required** I see that you disabled styling for the CSS files - Please, consider adding them back 🙏🏽 . These `linters ` help you write industry-standard code that communicates well with other developers. Here is a [read](https://webdesign.tutsplus.com/tutorials/how-to-lint-css-using-stylehint--cms-34422#:~:text=The%20use%20of%20CSS%20linters,static%20analysis%2C%20and%20so%20on.) to give you a glimpse as to why they're essential for your career as a developer.

File: index.html [hunk 2]
```diff
@@ -225,6 +226,210 @@ <h1 class="playfair">Get Started</h1>
...
+            <div class="grid-2-2">
+                <img class="img3" src="img/projects-dots-1.svg" alt="">
+            </div>
+            <div class="port-entry-1 grid-2-3" >                
+                <div class="poppins">
+                    <h1>Project name goes here</h1>
+                    <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javascript</li>
```
Comment (blessedjasonmwanza) [inline index.html:268]: - [ ] **Required:** Just a minor change here 🔹 😃 ... Kindly update the language links color (seems like you used black) to match your [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553) as per requirement 💪🏽 

## How your project looks 
image

# vs Figma design (#1A2236)
image

File: index.html [hunk 3]
```diff
@@ -225,6 +226,210 @@ <h1 class="playfair">Get Started</h1>
...
+                    </ul>
+                    <ul class="ul-2">
+                        
+                        <li>
+                            <label for="txt-msg">Message</label>
+                            <textarea name="user_message" id="txt-msg" class="text-msg" cols="30" rows="10" placeholder="Enter your message here..." maxlength="500" required></textarea>
+                        </li>
+                    </ul>
+                    <button type="submit" class="button1">Start collaboration</button>    
+                </form>
```
Comment (blessedjasonmwanza) [inline index.html:412]: - [ ] **Required** - To be honest, you're rocking on this project 🚀 . Only minor color changes to maintain the design are required. Kindly ensure you apply the dark color (the one used on the [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A553)) to the inputs, only when its in focus.

# Your current UI
image

# vs Design

image

**Review by blessedjasonmwanza — CHANGES_REQUESTED** [review]
> # Changes Required 
> ### Hi @Kwentsir
> Nice job! 🎉  I love your project and how its coming up 🎉 , you are almost there.
> 
> 
> 
> ## (Highlights) Good Points: 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check my inline comments.
> 
> ## [Optional] suggestions:
> * [ ]  Please check my inline comments.
> 
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear
> Please, remember to tag @topeogunleye in your question so I can receive the notification.
>
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by roshan-bajgain — APPROVED** [review]
Hi @lorenzoworx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**